recipes-devtools/gie/tensorrt-plugins_7.1.3-1.bb

```diff
@@ -0,0 +1,51 @@
+DESCRIPTION = "NVIDIA TensorRT Plugins for deep learning"
```

Comment:
Due to the dependency on protobuf, the recipe for which is in meta-oe, this recipe needs to be located under external/openembedded-layer (along with its patches), just as it is on the master branch.